src/blas/KokkosBlas2_gemv.hpp

```diff
@@ -139,11 +139,7 @@ gemv (const char trans[],
     typename YViewType::device_type,
     Kokkos::MemoryTraits<Kokkos::Unmanaged> > YVT;
 
-  if ((trans[0] == 'T' || trans[0] == 't') && A.extent(0) == 0
-#ifdef KOKKOS_ENABLE_CUDA
-      && !std::is_same<typename AViewType::device_type::execution_space, Kokkos::Cuda>::value
-#endif
-    )
+  if ((trans[0] == 'T' || trans[0] == 't' || trans[0] == 'C' || trans[0] == 'c' || trans[0] == 'H' || trans[0] == 'h'))
```

Comment:
Why "h" ? T is transpose and C is conj transpose. Does H mean ?

Reply:
Fixed with new push